masu/util/aws/region_map.py

```diff
@@ -0,0 +1,133 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Extracts information from AWS Regions And Endpoints page."""
+
+import json
+import logging
+import re
+
+import bs4
+import requests
+
+LOG = logging.getLogger(__name__)
+URL = 'https://docs.aws.amazon.com/general/latest/gr/rande.html'
```

Comment:
Would it be worth having an environment variable that we could override in case this page changes/goes away?

Reply:
This url hasn't changed in years. So, I'm pretty confident that there's a low probability that it'll change.

If it does change anytime soon, I think we can reevaluate then.